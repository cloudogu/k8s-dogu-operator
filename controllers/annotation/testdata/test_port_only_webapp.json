{
  "Env": [
    "SERVICE_80_TAGS=webapp"
  ],
  "ExposedPorts": {
    "80/tcp": {}
  }
}