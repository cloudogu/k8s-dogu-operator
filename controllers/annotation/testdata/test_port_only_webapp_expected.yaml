# This yaml contains a ces service annotation for the web app. Only the annotations are compared.
apiVersion: v1
kind: Service
metadata:
  name: serviceName
  annotations:
    k8s-dogu-operator.cloudogu.com/ces-services: '[{"name":"serviceName","port":80,"location":"/serviceName","pass":"/serviceName"}]'
spec:
  selector:
    name: serviceName
  ports:
    - protocol: TCP
      port: 80